common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -798,6 +798,58 @@ def test_export_to_xml_unicode_characters(self):
         self.assertEqual(xml.get('display_name'), u'\u8fd9\u662f\u6587')
 
 
+@ddt.ddt
+@patch.object(settings, 'FEATURES', create=True, new={
+    'FALLBACK_TO_ENGLISH_TRANSCRIPTS': False,
+})
+class VideoDescriptorStudentViewDataTestCase(unittest.TestCase):
+    """
+    Make sure that VideoDescriptor returns the expected student_view_data.
+    """
+
+    VIDEO_URL_1 = 'http://www.example.com/source_low.mp4'
+    VIDEO_URL_2 = 'http://www.example.com/source_med.mp4'
+    VIDEO_URL_3 = 'http://www.example.com/source_high.mp4'
+
+    @ddt.data(
```

Comment:
Niptuck: The `dict`, although being correct, is a bit confusing. Could you please use the braces `{}` syntax?

Reply:
Fixed https://github.com/edx/edx-platform/pull/16590/commits/879c979d49eab7c8f2773fe2f13c031234cc3e72#diff-71689a252fd5550e8c48c86bcfaa9f5dR815